### DISCUSSION TITLE
Developing a React Native Windows application

### DISCUSSION THREAD

**Discussion by AlmTalal** [software-mansion/react-native-reanimated#5657] [Q&A]
Hey ! Im developing a react-native-windows app and I want to add to it some animations. Im running trough some unexpected problems so before deeping to much I wanna know if this library support React Native Windows. Thnaks !

**Comment by piaskowyk** [comment]
Hey, we have partial support for react-native-windows. This means that we utilize a web JS-based implementation for Windows, but without threading support.
